### ISSUE TITLE
Support specifying system packages to install in Docker

### ISSUE THREAD

**Issue by palmskog** [coq-community/templates#41]
Some extraction projects rely on OCaml libraries that require certain system packages to be installed. There should be a way in `meta.yml` to specify these system packages and have them be installed in Travis and other CIs that rely on Docker images.

**Comment by Zimmi48** [comment]
The good practice in this case is to use opam depext fields. Then, the CI templates could simply run `opam-depext` (which is included in opam >= 1.1).

**Comment by erikmd** [comment]
@Zimmi48 for this feature request, should we create a related issue in https://github.com/coq-community/docker-coq-action/issues as well?

**Comment by Zimmi48** [comment]
@erikmd I guess so.

**Comment by erikmd** [comment]
Hi @palmskog,

do you believe this use case still needs some automation via a dedicated feature in docker-coq-action and/or in templates?

FWIW, there is now:
* some documentation in docker-coq-action's README: [Install Debian packages](https://github.com/coq-community/docker-coq-action#install-debian-packages);
* and ["demo" GHA job in docker-coq-action's CI](https://github.com/coq-community/docker-coq-action/blob/953471f78ec9e891784b2f8710445d1d43246e7f/.github/workflows/coq-demo.yml#L233-L273) that checks on a docker-coq image, some install script example using `apt-get`.

But as mentioned earlier by @Zimmi48 − https://github.com/coq-community/templates/issues/41#issuecomment-635166226 −, some (not all) system-wide install can be triggered by `opam-depext`…

**Comment by palmskog** [comment]
@erikmd I still need this feature for some of my projects, but the non-depext workaround (your installation of Debian packages) is not a huge pain. But hasn't depext now been improved in opam 2.1 and later? I think we can keep this issue open until we figure out the "right" way to install depext'd stuff.

**Comment by Zimmi48** [comment]
Yes, `opam-depext` is now superseded by `opam install` itself in opam 2.1.
